### PR TITLE
minicart on change 

### PR DESCRIPTION
**Pull request by perseusl**
-added onchange in qty of minicart
-fixes some javascript errors

### CODE REVIEW

File: app/design/frontend/rb/default/template/page/html/pager.phtml [hunk 1]
```diff
@@ -31,6 +31,10 @@
  * @see Mage_Page_Block_Html_Pager
  */
 ?>
+<?php
+    var_dump($this->getCollection()->getSize());
```
Comment (eshleebien) [inline app/design/frontend/rb/default/template/page/html/pager.phtml:35]: var_dump then die?

File: js/parallax/normal.js [hunk 2]
```diff
@@ -924,7 +924,7 @@ king_david.define('king_david-maps', function ($, _) {
   function initMaps() {
     $maps = $doc.find(namespace);
     if ($maps.length) {
-      king_david.script('https://maps.googleapis.com/maps/api/js?v=3.exp&sensor=false&callback=_wf_maps_loaded');
+//      king_david.script('https://maps.googleapis.com/maps/api/js?v=3.exp&sensor=false&callback=_wf_maps_loaded');
```
Comment (eshleebien) [inline js/parallax/normal.js:927]: intentional?

Comment (perseusl) [inline js/parallax/normal.js:927]: yes, this thing causes errors

File: app/design/frontend/rb/default/template/page/html/pager.phtml (hunk 1 above)
Comment (perseusl) [inline app/design/frontend/rb/default/template/page/html/pager.phtml:35]: no worries, this shit wasn't even called, dunno why, will fix it later

File: js/parallax/normal.js (hunk 2 above)
Comment (joncepeda) [inline js/parallax/normal.js:927]: So pro man.

File: app/design/frontend/rb/default/template/page/html/pager.phtml (hunk 1 above)
Comment (eshleebien) [inline app/design/frontend/rb/default/template/page/html/pager.phtml:35]: oke :+1:

File: js/parallax/normal.js (hunk 2 above)
Comment (eshleebien) [inline js/parallax/normal.js:927]: so how do we fix it?

Comment (perseusl) [inline js/parallax/normal.js:927]: it's already fixed by having it commented. it's just dual maps.google api declaration

**Pull request merged**